src/sign/CMakeLists.txt

```diff
@@ -6,4 +6,4 @@ find_library(cloudhsmpkcs11 STATIC)
 add_executable(sign ec_sign.c rsa_sign.c sign.c common.c sign.h)
 target_link_libraries(sign cloudhsmpkcs11)
 
-add_test(sign sign --pin ${HSM_USER}:${HSM_PASSWORD})
+# add_test(sign sign --pin ${HSM_USER}:${HSM_PASSWORD})
```

Comment:
I need a refresher, should we fix the test instead of removing it? Or was there another reason this test was bad?

Reply:
Instead of removing the test, can you replace the curve with 

```
CK_BYTE secp256k1[] = {0x06, 0x08, 0x2a, 0x86, 0x48, 0xce, 0x3d, 0x03, 0x01, 0x07};
```
and update the comment with 
```
openssl ecparam -name prime256v1 -outform DER | hexdump -C
```

This way RHEL6 will pass.